src/context/alert.tsx

```diff
@@ -161,6 +162,8 @@ const messageToTranslationKeyMappings: Record<string, string> = {
   [ERROR_MESSAGE.PAYMENT_QR_UNSUPPORTED]: "unsupportedPaymentMethodInPaymentQR",
   [ERROR_MESSAGE.PAYMENT_QR_UNSUPPORTED_TEXT_DISABLED]:
     "unsupportedPaymentMethodInPaymentQRTextDisabled",
+  [ERROR_MESSAGE.GOVWALLET_ACCOUNT_DEACTIVATED_ERROR]:
```

Comment:
This allows `getTranslationKeyFromErrorMessage` to resolve our error dialog message to this constant. The error dialog will be shown with the default title, and this error message.

Pending input from Eida on whether the default title is fine.